tooltester/src/main/java/io/dockstore/tooltester/client/cli/Report.java

```diff
@@ -0,0 +1,60 @@
+package io.dockstore.tooltester.client.cli;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author gluu
+ * @since 23/01/17
+ */
+class Report {
+    private static final char COMMA_SEPARATOR = ',';
+    //private static final char TAB_SEPERATOR = '\t';
+    private static final List<String> HEADER = Arrays.asList("ID", "DATE", "Tool/Workflow ID", "Version", "Location of testing", "Parameter file", "Runtime", "Status of Test Files", "Status of Dockerfiles");
+
+    private Writer writer;
+
+    Report(String name) {
+
+        try {
+            this.writer = new OutputStreamWriter(new FileOutputStream(name), StandardCharsets.UTF_8);
+            writeLine(HEADER);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    void writeLine(List<String> values){
```

Comment:
This implementation could be slow and memory intensive since you build the stringbuilder from values (storing the data twice). Probably best just to write to the file as-you-go.